docker-compose.yml

```diff
@@ -115,6 +147,34 @@ services:
       - ./src/blocks:/blocks
       - blocks-node-modules:/blocks/node_modules
 
+  selenium:
+    image: ${SELENIUM_IMAGE:-selenium/standalone-chrome:4.8.3}
+    # image: ${SELENIUM_IMAGE:-seleniarm/standalone-chromium:4.8.3} # use this for ARM64
+    shm_size: '1gb'
+    ports:
+      # Increment default port numbers by 1 to avoid conflict with organic dev.
+      - 4445:4444
+
+      # https://github.com/seleniumhq-community/docker-seleniarm#running-the-arm-container-images
+      # Visit http://localhost:7901 to see what's happening in the browser
+      # during selenium tests
+      #
+      # !!! VNC password is "secret" !!!
+      - 5901:5900  # port for external VNC client
+      - 7901:7900  # noVNC.com UI
+
+  nginx-proxy:
+    image: nginxproxy/nginx-proxy:1642
```

Comment:
I used the same image as in organic-dev for consistency.

Reply:
Although I didn't do this for the selenium image. I thought having an updated selenium was more important.